weaknesses/MASVS-CRYPTO/MASWE-0025.md

```diff
@@ -14,10 +14,26 @@ refs:
 - https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-131Ar2.pdf
 - https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-5.pdf
 - https://csrc.nist.gov/pubs/ir/8547/ipd
-draft:
-  description: The use of algorithms with insufficient strength for signatures such as SHA1withRSA, etc. in a security-sensitive context should be avoided to ensure the integrity and authenticity of the data.
-  topics: null
-status: placeholder
+status: new
 
 ---
 
+## Overview
+
+Using deprecated, risky or broken algorithms for the purpose of ensuring accountability and non-reputation through the use of signatures could make it possible for an attacker to execute digital signature forgery and compromise the integrity and authenticity of the data at rest and in transit.
+
+## Impact
+
+- **Loss of Integrity and authenticity**: Signature forgery may allow the attacker to compromise the integrity and authenticity of the data by signing the data on behalf of another entity.
+- **Loss of accountability**: Signature forgery allows for plausible deniability and diminishes accountability.
```

Comment:
To have accountability, we usually need identification, plus logging of actions. Non-repudiation seems more to the point here.

```suggestion
- **Loss of non-repudiation**: Signature forgery allows for plausible deniability and diminishes accountability.
```